### DISCUSSION TITLE
Is there a way to use the inpainting model independantly?

### DISCUSSION THREAD

**Discussion by diagonalge** [lllyasviel/Fooocus#2623] [Q&A]
Is there a script/method to run the inpainting model independently with diffusers for example? Any help in this regard would be appreciated

**Comment by mashb1t** [comment]
You may be able to use ComfyUI for this (https://comfyanonymous.github.io/ComfyUI_examples/inpaint/) but applying the Fooocus inpainting model as LoRA on the base model. Please find the inpainting model details in the config.py of Fooocus.
